### DISCUSSION TITLE
Plus Server Config for Older NDI Polaris Hybrid?

### DISCUSSION THREAD

**Discussion by treelinemike** [PlusToolkit/PlusLib#808] [Q&A] [answered]
This may be a long shot, but does anyone have experience running an older NDI Polaris Hybrid optical tracker (c. 2006) via the Plus server? I tried this with the included NDICAPI drivers but it looks like they send some commands that our Polaris doesn't support. Figured I'd check here first before getting too deep into the weeds of debugging and implementing a new one. Maybe I'm just missing a config option? We currently run these trackers with passive markers only using our own standalone MATLAB implementation of the NDI API and are now looking to integrate with 3D Slicer.

Thanks in advance!
Mike Kokko (Dartmouth)

**Comment by lassoan** [comment, marked as answer]
You can register at https://support.ndigital.com/ and see if they have firmware updates. We were able to update our trackers that were purchased around 2009.

If that does not work then you can check exactly what commands fail and if they can be skipped or replaced with alternative commands.

**Reply by treelinemike** [reply]
Excellent - thanks so much! I had been hesitant to update the firmware for fear of bricking the trackers, but knowing that it worked for you I tried it and am able to track tools in Slicer now. Really appreciate the advice.
